salt/states/boto_ec2.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+'''
+Manage EC2
+
+.. versionadded:: TBD
+This module provides an interface to the Elastic Compute Cloud (EC2) service
+from AWS.
+
+The below code creates a key pair:
+
+.. code-block:: yaml
+
+    create-key-pair:
+      boto_ec2.present:
+        - name: mykeypair
+        - save_priv: /root/
+        - region: eu-west-1
+        - keyid: GKTADJGHEIQSXMKKRBJ08H
+        - key: askdjghsdfjkghWupUjasdflkdfklgjsdfjajkghs
+
+.. code-block:: yaml
+
+    import-key-pair:
+       boto_ec2.present:
+        - name: mykeypair
+        - upload_pub: 'ssh-rsa AAAA'
+        - keyid: GKTADJGHEIQSXMKKRBJ08H
+        - key: askdjghsdfjkghWupUjasdflkdfklgjsdfjajkghs
+The below code deleted a key pair:
+
+.. code-block:: yaml
+    delete-key-pair:
+      boto_ec2.absent:
+        - name: mykeypair9
+        - region: eu-west-1
+        - keyid: GKTADJGHEIQSXMKKRBJ08H
+        - key: askdjghsdfjkghWupUjasdflkdfklgjsdfjajkghs
+'''
+import logging
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    '''
+    Only load if boto is available.
+    '''
+    if 'boto_ec2.get_key' in __salt__:
+        return 'boto_ec2'
+    else:
+        return False
+
+
+def present(name, save_priv=None, upload_pub=None, region=None, key=None,
```

Comment:
So it is `save_priv` or `save_private` (`upload_pub` vs `upload_public`)? I would like to have it less cryptic and better explained. Could you please expand these parameters?

Thanks in advance!